tests/e2e/utils/load_balancer/setup_load_balancer.py

```diff
@@ -0,0 +1,262 @@
+        env_dict={
+            "certArn": tls_cert_arn,
+        },
+    )
+
+
+# Step 4: Configure load balancer controller manifests and create an IAM role for controller service account
+def configure_load_balancer_controller(
+    region: str, cluster_name: str, policy_name: str = None
+) -> Tuple[str, str]:
+    policy_name = policy_name or rand_name(f"alb_ingress_controller_{cluster_name}")
+    ec2_client = get_ec2_client(region)
+    eks_client = get_eks_client(region)
+
+    # create an iam service account with required permissions for the controller
+    cluster.associate_iam_oidc_provider(cluster_name, region)
+    alb_policy = IAMPolicy(name=policy_name, region=region)
+    alb_policy.create(
+        policy_document=load_json_file(
+            "../../awsconfigs/infra_configs/iam_alb_ingress_policy.json"
+        )
+    )
+
+    alb_sa_name = "alb-ingress-controller"
+    alb_sa_namespace = "kubeflow"
+    cluster.create_iam_service_account(
+        alb_sa_name, alb_sa_namespace, cluster_name, region, [alb_policy.arn]
+    )
+
+    # tag cluster subnet with kubernetes.io/cluster/<cluster_name> tag
+    # see prerequisites in https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html
+    cluster_desc = eks_client.describe_cluster(name=cluster_name)
+    ec2_client.create_tags(
+        Resources=cluster_desc["cluster"]["resourcesVpcConfig"]["subnetIds"],
+        Tags=[
+            {"Key": f"kubernetes.io/cluster/{cluster_name}", "Value": "shared"},
+        ],
+    )
+
+    # substitute the cluster_name for the load balancer controller
+    configure_env_file(
+        env_file_path="../../awsconfigs/common/aws-alb-ingress-controller/base/params.env",
+        env_dict={
+            "clusterName": cluster_name,
+        },
+    )
+
+    return {
+        "serviceAccount": {
+            "name": alb_sa_name,
+            "namespace": alb_sa_namespace,
+            "policyArn": alb_policy.arn,
+        }
+    }
+
+
+def wait_for_alb_dns(cluster_name: str, region: str):
+    logger.info("waiting for ALB creation ...")
+
+    def callback():
+        ingress = get_ingress(cluster_name, region)
+
+        assert ingress.get("status") is not None
+        assert ingress["status"]["loadBalancer"] is not None
+        assert len(ingress["status"]["loadBalancer"]["ingress"]) > 0
+        assert (
+            ingress["status"]["loadBalancer"]["ingress"][0].get("hostname", None)
+            is not None
+        )
+
+    wait_for(callback)
+
+
+def wait_for_alb_status(alb_dns: str, region: str, expected_status: str = "active"):
+    logger.info(f" {alb_dns} waiting for ALB status = {expected_status} ...")
+
+    alb = ElasticLoadBalancingV2(dns=alb_dns, region=region)
+    def callback():
+        assert alb.describe()["State"]["Code"] == expected_status
+
+    wait_for(callback)
+
+
+def create_ingress():
+    kubectl_apply_kustomize(path=common.LB_KUSTOMIZE_PATH)
```

Comment:
where do i see this file ? the only file in path `LB_KUSTOMIZE_PATH = "../../docs/deployment/add-ons/load-balancer/"` is README.md am I missing something here?

Reply:
https://github.com/surajkota/kubeflow-manifests/blob/lb_dex/docs/deployment/add-ons/load-balancer/kustomization.yaml


its part of this PR, can you check again?